apps/ui/docs/FormAndDataFetching.md

```diff
@@ -0,0 +1,85 @@
+# Form and Data Fetching
+
+Libraries and bindings:
+
+- [Zod validation](https://zod.dev/)
+- [React-Hook-Form](https://react-hook-form.com/)
+  - [antd-zod](https://github.com/MrBr/antd-zod) (changes zod schema into antd rules)
+- [react-query](https://tanstack.com/query/v5/docs/framework/react/overview)
+
+(Already Included)
+
+- [Antd Forms](https://ant.design/components/form)
```

Comment:
I think you can remove `(Already Included)` and group this with the rest of the links